Proofed en_US base.po, suggestions & alternative lines

freshreplicant opened the pull request:
I took the current dialogue from the en_US version of base.po and went through it, mainly fixing punctuation, one typo and making a few suggestions for alternative lines. 

There are also a few more general suggestions, such as in line 1333, 1444, 1479, 1492-1496, 1529, 1539, 1593, 1603-1604, 1613, 1616, 1622, etc. I was not sure if adding these in via comments prefaced by # was the right way to do this, please correct me if that was the wrong approach!

The renaming NPCs currently say the same thing, even if you choose not to go ahead with name changes (1862, 1868).

bitcraft commented on mods/tuxemon/l18n/en_US/LC_MESSAGES/base.po:
```diff
@@ -1439,15 +1441,15 @@ msgstr "I have been observing you for a while now, I'm glad my followers managed
 "....  .....  ....\n"
 "${{name}}?\n"
-"That sounds like a good name for someone who illegally owns a Tuxemon.\n"
+"That sounds like a good name for someone who illegally owns a Tuxemon.\n" # Why would it be a 'good name' for this?
```
Again, please don't add comments to the files like this.  If you have a question or suggestion about a specific dialog line, open the file in the GitHub file viewer, click the line number, then copy/paste the url (which will be linked to that line) and either bring it up in the discord or open a github issue and provide that link.  Its possible that these comments would be missed in code review, leading to "pollution" of comments.  Comments in a translation file should only provide context to the string.  Questions or suggestions don't belong.

bitcraft commented on mods/tuxemon/l18n/en_US/LC_MESSAGES/base.po:
```diff
@@ -1487,27 +1489,32 @@ msgid "hellothere"
 msgid "kmere"
-msgstr "Why don't you come over here and talk to me?"
+msgstr "Why don't you come over here and talk to me?" # Why does Aeble wish to talk to the player? 
+# The dialogue that follows is great, but maybe there’s a way to make it feel less obviously like exposition,
```
Please make sure these comments are removed and discussions opened in another format: forums/github issues/or discord.

bitcraft reviewed (changes requested):
- The punctuation and grammar fixes are welcomed.
- Please don't change msgids
- Please don't add comments unless they provide context to the string.  Comments should be avoided.

Thanks for your interest in the dialogs.  The comments, questions, and proposals should be removed.  In the future, these items should be discussed outside of the PR -- don't add them in order to discuss -- link the lines in an issue or forum post (or discord), then make the changes to the string itself, not a comment.

freshreplicant commented:
Hi @bitcraft thanks very much for taking the time to review my pull request and for all your pointers, much appreciated. They were very helpful and instructive. My most recent commit should address the issues you raised.

I have removed punctuation for the dialogue options, fixed the msgid and  as you suggested I removed all the comments and will raise those points for discussion elsewhere.

Qiangong2 commented on mods/tuxemon/l18n/en_US/LC_MESSAGES/base.po:
```diff
@@ -1397,7 +1398,8 @@ msgid "professor_dialog6.3"
 msgid "professor_dialog7"
-msgstr "Professor: I hope to see you around! I'm about to head on a journey, so maybe we'll meet up sometime."
+msgstr "Professor: Well that’s really all I have time for! I’m late heading out for an expedition,\n"
+but I will be in touch to see how you are getting along."
```
This is being redone at the moment and will be deleted from the base.po
